File: python/src/items/Sulfuras.py
from src.Items.NormalItem import *

'''
This class will be for Sulfuras, legendary item.
'''

class Sulfuras(NormalItem):

    def __init__(self, name, sell_in, quality):

        Item.__init__(self, name, sell_in, quality)

    # Override new updateQuality method
    def updateQuality(self):
        
        self.quality = 80

File: java/domain/Sulfuras.java
package java.domain;

public class Sulfuras extends NormalItem{
    

    public Sulfuras(String name, int sellIn, int quality){
    
        super(name, sellIn, quality);
    }

    @Override
    public void updateQuality(){

        this.setQuality(80);
    }
}
